include/gzip/utils.hpp

```diff
@@ -1,4 +1,7 @@
+#include <cstdlib>
```

Comment:
Read http://www.cplusplus.com/reference/cstdlib/, but still not sure why this header is needed here

Reply:
ah, seeing https://github.com/mapbox/gzip-hpp/pull/19#discussion_r144605991

Reply:
Needed for `uint8_t`. Without it, this header would not compile on its own. So this is an `IWYU` fix, aka `include what you use`.

Reply:
More details on that at https://github.com/include-what-you-use/include-what-you-use. `clang-tidy` can also flag when headers are depending on types but not including the header for them.

Reply:
Is include-what-you-use automatically included in clang-tidy? Or are you using it here as a general good practice?

Also,  is tidy succeeding for you locally? I'm getting errors:

```
/Users/carol/dev/gzip-hpp/mason_packages/osx-x86_64/benchmark/1.2.0/include/benchmark/benchmark.h:832:3: error: Potential memory leak [clang-analyzer-cplusplus.NewDeleteLeaks,-warnings-as-errors]
  return internal::RegisterBenchmarkInternal(
  ^
/Users/carol/dev/gzip-hpp/bench/run.cpp:47:5: note: Taking false branch
    if (!stream.is_open())
    ^
Suppressed 10135 warnings (10134 in non-user code, 1 NOLINT).
Use -header-filter=.* to display errors from all non-system headers. Use -system-headers to display errors from system headers as well.
1 warning treated as error
21620 warnings generated.
/Users/carol/dev/gzip-hpp/test/test.cpp:28:5: error: consider replacing 'unsigned long' with 'uint64' [google-runtime-int,-warnings-as-errors]
    unsigned long l = 2000000001;
    ^
Suppressed 21619 warnings (21619 in non-user code).
Use -header-filter=.* to display errors from all non-system headers. Use -system-headers to display errors from system headers as well.
1 warning treated as error
Applying fixes ...
```